src/seedu/addressbook/data/person/Street.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+/**
+ * represents the street name of address
+ * Guarantees: immutable
+ */
+public class Street {
+	
+	private String value;
+	
+	/**
+	 * Constructor of Street
+	 * @param streetName
+	 */
+	public Street(String streetName) {
+		this.value = streetName;
+	}
+	
+	/**
+	 * Get the value of street name
+	 */
+	public String getStreetName() {
+		return value;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.